pkg/disk/partition_table.go

```diff
@@ -160,6 +161,289 @@ func NewPartitionTable(basePT *PartitionTable, mountpoints []blueprint.Filesyste
 	return newPT, nil
 }
 
+func mkBIOSBoot() Partition {
+	return Partition{
+		Size:     1 * common.MebiByte,
+		Bootable: true,
+		Type:     BIOSBootPartitionGUID,
+		UUID:     BIOSBootPartitionUUID,
+	}
+}
+
+func mkESP(size uint64) Partition {
```

Comment:
(for later) we should make a note that we use this helper in `otk-gen-partition-table` somehow, would be nice to consolidate this (similar to mkBIOSBoot)